zebra-chain/src/orchard/shielded_data.rs

```diff
@@ -155,8 +155,12 @@ impl ZcashSerialize for Flags {
 
```

Comment:
Can you also add the consensus rule to the `Flags` struct documentation?

We should also document that `bitflags` restricts its values to the set of valid flags, and link to
https://docs.rs/bitflags/1.2.1/bitflags/index.html

After you change the documentation, you can remove the `// Reserved, zeros (bits 2 .. 7)` comment.